.github/workflows/ci.yml

```diff
@@ -42,7 +48,7 @@ jobs:
       # Run the tests, but not if we're just gonna do coveralls later anyway
     - name: Run Tap tests
       if: matrix.platform.os != 'ubuntu-latest' || matrix.node-version != '12.x'
-      run: node . test -- -t600 -Rbase -c
+      run: node . run --ignore-scripts ${{ matrix.script }} -- -t600 -Rbase -c
```

Comment:
I do wonder if:

https://github.com/npm/cli/blob/e815cd4f79dafcc8fa1b8b2bec4c9b4acac89175/package.json#L210

is required (or even makes any difference) on CI, given that this change make it so that we're not running pre/post scripts on every test run, this `prelint` script is also not going to run on every test anymore

Reply:
hmm, so `--ignore-scripts` applies to everything and not just the top-level run-script?

The intention is to ignore pretest/posttest, but no other scripts. If that's what this does, then the only other alternative is to make some kind of `tests-only` script, and it means the npm 7 improvements to `--ignore-scripts` aren't actually useful :-/

Reply:
altho either way, clearing the cache first shouldn't matter on CI which starts fresh.